.github/workflows/sanity.yml

```diff
@@ -194,7 +218,7 @@ jobs:
 
```

Comment:
`publish-image` job needs to depend on the `unit tests` and `ensure clean branch` checks, so that we publish onyl if all jobs are successful (when building from main on a merge)

Reply:
Done